run_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+COVERAGE_COMMAND="coverage run --rcfile=../coverage-config --source='.' manage.py test --settings=concent_api.settings.testing"
+TEST_RUNNER_EXTRA_ARGUMENTS=""
```

Comment:
Well... that's not really what I had in mind. Now the command is in two places which is arguably worse because they'll definitely get out of sync sooner or later.

In my opinion it's enough to print the extra arguments. We don't need to see the full command in the output (it's in the script anyway). That's if you really want to print it - because just printing them does not guarantee that they had any effect.

Reply:
Ok, if you say so. In my opinion it is more convenient to see what you are actually executing. But I'll remove the print and the variable